<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-17T12:58:20 AEST" name="Default suite" finished-at="2025-04-17T12:59:11 AEST" duration-ms="50204">
    <groups>
    </groups>
    <test started-at="2025-04-17T12:58:20 AEST" name="Default test" finished-at="2025-04-17T12:59:11 AEST" duration-ms="50204">
      <class name="com.ecommerce.tests.common.cart.CartTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.ecommerce.tests.common.cart.CartTest@54422e18]" started-at="2025-04-17T12:58:20 AEST" name="setup" finished-at="2025-04-17T12:58:36 AEST" duration-ms="15237" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="initialize()[pri:0, instance:com.ecommerce.tests.common.cart.CartTest@54422e18]" started-at="2025-04-17T12:58:36 AEST" name="initialize" finished-at="2025-04-17T12:58:54 AEST" duration-ms="18648" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="testAddMultipleProductsToCart()[pri:0, instance:com.ecommerce.tests.common.cart.CartTest@54422e18]" started-at="2025-04-17T12:58:54 AEST" name="testAddMultipleProductsToCart" finished-at="2025-04-17T12:59:10 AEST" duration-ms="15932" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Product 'Men  Tshirt' not found in cart! ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Product 'Men  Tshirt' not found in cart! 
at com.ecommerce.components.ProductInCartComponent.getProductInCartInfo(ProductInCartComponent.java:39)
at com.ecommerce.pages.CartPage.getProductInCartInfo(CartPage.java:59)
at com.ecommerce.tests.common.cart.CartTest.testAddMultipleProductsToCart(CartTest.java:141)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddMultipleProductsToCart -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.ecommerce.tests.common.cart.CartTest@54422e18]" started-at="2025-04-17T12:59:10 AEST" name="teardown" finished-at="2025-04-17T12:59:11 AEST" duration-ms="318" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.ecommerce.tests.common.cart.CartTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
